cypress/integration/Order.spec.js

```diff
@@ -0,0 +1,14 @@
+/// <reference types="Cypress" />
+
+import { visitWithWeb3, tid, revertToSnapshot } from "../utils";
+
+context("Order", () => {
+  beforeEach(() => visitWithWeb3())
+  afterEach(() => revertToSnapshot());
+
+  it("should buy dai", () => {
```

Comment:
Just out of curiosity - what the reason behind this test?

Reply:
it was supposed to be just a skeleton for next test, i will remove it since its not doing anything and i think you did some work on other branch

Reply:
Yeah, I almost had a whole test regarding selling ETH but it fails with the error I gave you earlier  - `VM execution error - no revert` or something like this